js/navigation.js

```diff
@@ -192,5 +197,11 @@ favResultsDiv.addEventListener('click', function (event) {
       var shoppingCart = search.getResult(uri);
       shoppingList.addRecipe(shoppingCart);
     }
+  } else if (event.target.matches('p')) {
+    var uri = event.target.getAttribute('data-recipe-uri');
+    var shopRecipe = shoppingList.getRecipe(uri);
+    if (shopRecipe !== undefined) {
+      window.open(shopRecipe.recipe.url);
+    }
```

Comment:
Again, this would be much easier with an anchor tag, why is `window.open` on a `p` element necessary?